en/docs/testing/test-rest-endpoints-via-the-openapi-console.md

```diff
@@ -0,0 +1,32 @@
+# Test REST Endpoints via the OpenAPI Console
+
+Devant offers an integrated OpenAPI Console to test REST endpoints for the Service components you create and deploy. Since Devant secures REST APIs with OAuth 2.0 authentication, the OpenAPI Console generates test keys to help you test your APIs.
+
+Follow these steps to test a REST endpoint using the OpenAPI Console:
+
+1. Go to the [Devant Console](https://console.devant.dev/) and log in.
+2. Select the project.
+3. In the **Integrations** pane, click on the component you want to test.
+4. Click **Test** in the left navigation menu, then select **Console**. This will open the **OpenAPI Console** pane.
+5. In the **OpenAPI Console** pane, select the desired environment from the drop-down menu.
+6. Choose the endpoint you want to test from the **Endpoint** list.
+
+    !!! note
+        - The **Network Visibility** of an endpoint is configured in the **component.yaml** file. You can set it to **Public**, **Organization**, or **Project**.
+        - If the **Network Visibility** is set to **Public**, skip to step 8.
+        - If the **Network Visibility** is set to **Organization**, the endpoint is not publicly accessible. However, Devant allows you to generate a temporary test URL for testing purposes. This URL is publicly accessible (secured with OAuth 2.0) and remains active for 15 minutes. To generate the temporary URL, proceed to step 6.
+
+7. Generate a temporary test URL (if applicable):  
+   If the **Network Visibility** is set to **Organization**, click **Generate URL** to create a temporary test URL. This URL will be active for 15 minutes. If the visibility is not set to **Organization**, skip this step.
```

Comment:
Let's not reiterate the same info here. May be we can say the expiry duration here instead in the prev step

If Network Visibility is set to Organization, click Generate URL to create a temporary test URL, valid for 15 minutes. If not, skip this step.

Reply:
Fixed via [4dcea89](https://github.com/wso2/docs-devant/pull/59/commits/4dcea89c85fbc5f21aa7769a876bdc0bb3e7ab60)